.github/workflows/build.yml

```diff
@@ -33,6 +33,11 @@ jobs:
       uses: nttld/setup-ndk@v1
       with:
         ndk-version: r21e
+    - name: Setup Emscripten
+      if: startsWith(matrix.os, 'ubuntu')
+      uses: mymindstorm/setup-emsdk@v11
+      with:
+        version: 2.0.23 # NB: Matches .NET 6
```

Comment:
Could alternatively follow [the instructions here](https://emscripten.org/docs/getting_started/downloads.html). This just seemed easier, and you were already using a custom action for the NDK.